### PR TITLE
Fix 'coroutine 'main' was never awaited' message when run from CLI

### PR DESCRIPTION
**Pull request by olalid**
This update creates a separate sync main function that can be called from a sync script and renames the old async version to async_main.

### CODE REVIEW

File: pyeasee/__main__.py [hunk 1]
```diff
@@ -335,10 +335,14 @@ def str_fixed_length(myStr, length: int):
...
+def main():
+    asyncio.run(async_main())
+
+
 if __name__ == "__main__":
     import time
 
     s = time.perf_counter()
-    asyncio.run(main())
+    asyncio.run(async_main())
```
Comment (fondberg) [inline pyeasee/__main__.py:346]: Best way is to send in the event loop from the context. I'll share an example tomorrow

Comment (Hellowlol) [inline pyeasee/__main__.py:346]: Should this just be main()?

Comment (olalid) [reply pyeasee/__main__.py:346]: Ok, sorry, now I understand what you mean. Perhaps :)

Comment (olalid) [reply pyeasee/__main__.py]: Changed it.

**Pull request merged**